news/2024/2024-09-29-community-mentorship-program-autumn-2024-signups-now-open.md

```diff
@@ -0,0 +1,33 @@
+---
+layout: post
+title: Community Mentorship Program Winter 2024 Signups Now Open
+date: 2024-09-29 10:00:00 +0000
+---
+
+The 28th cycle of the Community Mentorship program is upon us! Get ready to delve deep into the world of mapping, modding, or storyboarding.
+
+![](/wiki/shared/news/banners/community-mentorship-program.jpg)
+
+The Community Mentorship program serves as a place for upcoming mappers, modders, and storyboarders to develop their skills under the guidance of more experienced members in the community. If you've ever found yourself wanting to expand your current knowledge, this might just be the opportunity you need.
+
+Over the course of a cycle, you'll have the opportunity to learn from your mentor, participate in various mentorship events such as feedback lobbies and speedmapping contests, organise guest difficulties for your mapsets on our job board, and more! General mapping discussion, feedback channels and the guest difficulty job board are now open for anyone to contribute, so if this sounds interesting to you, [join our Discord server to take part](https://discord.gg/Ft2FtXmBgx).
```

Comment:
```suggestion
Over the course of a cycle, you'll have the opportunity to learn from your mentor, participate in various mentorship events such as feedback lobbies and speedmapping contests, organize guest difficulties for your mapsets on our job board, and more! General mapping discussion, feedback channels and the guest difficulty job board are now open for anyone to contribute, so if this sounds interesting to you, [join our Discord server to take part](https://discord.gg/Ft2FtXmBgx).
```

Since Program is american english it makes sense to use the american word organize vs the british organise.

Reply:
"program" is just part of a name here. it wouldn't be changed to "Community Mentorship Programme" just because the article is in british english